server/services/contactsService.test.ts

```diff
@@ -441,4 +446,54 @@ describe('contactsService', () => {
       )
     })
   })
+
+  describe('getLanguageReference', () => {
+    it('Should get the language reference', async () => {
+      // Given
+      const expectedLanguage: Language = {
+        languageId: 23,
+        nomisCode: 'ENG',
+        nomisDescription: 'English',
+        isoAlpha2: 'en',
+        isoAlpha3: 'eng',
+        isoLanguageDesc: 'English',
+        displaySequence: 1,
+      }
+
+      // When
+      apiClient.getLanguageReference.mockResolvedValue(expectedLanguage)
+      const contact = await service.getLanguageReference(user)
+
+      // Then
+      expect(contact).toStrictEqual(expectedLanguage)
+      expect(apiClient.getLanguageReference).toHaveBeenCalledWith(user)
+    })
+
+    it('Propagates errors', async () => {
+      apiClient.getLanguageReference.mockRejectedValue(new Error('some error'))
+      await expect(apiClient.getLanguageReference(user)).rejects.toEqual(new Error('some error'))
+    })
+  })
+
+  describe('updateContactById', () => {
+    // Given
+    const request: PatchContactRequest = {
+      languageCode: 'ENG',
+      updatedBy: 'user1',
+    }
+    it('Should get the language reference', async () => {
```

Comment:
copy past issue, this needs updating , `Should update the language reference`?

Reply:
test name corrected